examples/RFQuack-esp32-CC1101-serial/src/main.cpp

```diff
@@ -31,16 +24,38 @@
 // frequency deviation:                 48.0 kHz
 // sync word:                           0xD391
 
+
+/* Radio configuration */
+#include <radio/drivers.h>
+
+#define USE_RADIOA
+typedef RFQCC1101 RadioA;
+RadioA radioA = new Module(2, 21, NC);
+
+// #define USE_RADIOB
+typedef NoRadio RadioB;
+RadioB radioB;
+
+// #define USE_RADIOC
+typedef NoRadio RadioC;
+RadioC radioC;
+
+// #define USE_RADIOD
+typedef NoRadio RadioD;
+RadioD radioD;
+
+// #define USE_RADIOE
+typedef NoRadio RadioE;
+RadioE radioE;
+
 /*****************************************************************************
  * /RFQuack configuration - DO NOT EDIT BELOW THIS LINE
  *****************************************************************************/
- #include "rfquack.h"
-
- RadioA radio = new Module(RFQUACK_RADIO_PIN_CS, RFQUACK_RADIO_PIN_GDO0, RFQUACK_RADIO_PIN_GDO1);
+#include <rfquack.h>
 
- void setup() {
-   rfquack_setup(radio);
- }
+void setup() {
+  rfquack_setup(radioA, radioB, radioC, radioD, radioE);
```

Comment:
Might be a very stupid question, but please be patient :-)

Remind me why we can't do:

`rfquack_setup(radioA, radioB, null, null, null)`

Reply:
You are right, we can do that!

I thought it was easier to leave there all the 5 radios in order to ease the plug-in of a new radio module.
But, as you suggest, the common use case is 1/2 radios; it's misleading to have all of them there.

Please, check the new commit :smile:

Reply:
.... the common use case **IS** 2 radios :sweat_smile:
Is it better to default to two radios (radioA, radioB)? Should I add a  third one (radioC) commented?

Reply:
It doesn't really matter. You can mix and match. As long as the users understand that they have the "liberty" to choose, it's fine to me.